controllers/authors.js

```diff
@@ -10,12 +11,13 @@ const getAllAuthors = async (req, res) => {
   }
 }
 
+
+
 const getAuthorById = async (req, res) => {
   try {
     const { id } = req.params
-console.log(id)
     const author = await models.Authors.findOne({
-      where: { id },
+      where: { id: {[models.Op.like]: `%${id}` } },
```

Comment:
I changed your function to this to get it to work

const { identifier } = request.params
  const author = await models.Authors.findOne({    where: {      [models.Sequelize.Op.or]: [        { id: identifier },        { nameLast: { [models.Sequelize.Op.like]: `%${identifier}%` } },      ]    },    include: [{      model: models.Novels,      include: [{ model: models.Genres }]    }]  })
  return author    ? response.send(author)    : response.sendStatus(404)}